airflow-core/src/airflow/ui/src/pages/Events/EventsFilters.tsx

```diff
@@ -0,0 +1,226 @@
+  const afterFilter = searchParams.get(AFTER_PARAM) ?? "";
+  const beforeFilter = searchParams.get(BEFORE_PARAM) ?? "";
+  const dagIdFilter = searchParams.get(DAG_ID_PARAM) ?? "";
+  const eventTypeFilter = searchParams.get(EVENT_TYPE_PARAM) ?? "";
+  const mapIndexFilter = searchParams.get(MAP_INDEX_PARAM) ?? "";
+  const runIdFilter = searchParams.get(RUN_ID_PARAM) ?? "";
+  const taskIdFilter = searchParams.get(TASK_ID_PARAM) ?? "";
+  const tryNumberFilter = searchParams.get(TRY_NUMBER_PARAM) ?? "";
+  const userFilter = searchParams.get(USER_PARAM) ?? "";
+
+  const handleInputChange = useCallback(
+    (paramName: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
+      const { value } = event.target;
+
+      if (value) {
+        searchParams.set(paramName, value);
+      } else {
+        searchParams.delete(paramName);
+      }
+      resetPagination();
+      setSearchParams(searchParams);
+    },
+    [resetPagination, searchParams, setSearchParams],
+  );
+
+  const handleDateTimeChange = useCallback(
+    (paramName: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
+      const { value } = event.target;
+
+      if (value) {
+        // Convert datetime-local value to ISO string
+        try {
+          const date = new Date(value);
+
+          if (!isNaN(date.getTime())) {
+            searchParams.set(paramName, date.toISOString());
+          }
+        } catch {
+          // Invalid date, remove the parameter
+          searchParams.delete(paramName);
+        }
+      } else {
+        searchParams.delete(paramName);
+      }
+      resetPagination();
+      setSearchParams(searchParams);
+    },
+    [resetPagination, searchParams, setSearchParams],
+  );
+
+  const handleClearFilters = useCallback(() => {
+    searchParams.delete(AFTER_PARAM);
+    searchParams.delete(BEFORE_PARAM);
+    searchParams.delete(DAG_ID_PARAM);
+    searchParams.delete(EVENT_TYPE_PARAM);
+    searchParams.delete(MAP_INDEX_PARAM);
+    searchParams.delete(RUN_ID_PARAM);
+    searchParams.delete(TASK_ID_PARAM);
+    searchParams.delete(TRY_NUMBER_PARAM);
+    searchParams.delete(USER_PARAM);
+
+    resetPagination();
+    setSearchParams(searchParams);
+  }, [resetPagination, searchParams, setSearchParams]);
+
+  const filterCount = getFilterCount({
+    after: afterFilter,
+    before: beforeFilter,
+    dagId: dagIdFilter,
+    eventType: eventTypeFilter,
+    mapIndex: mapIndexFilter,
+    runId: runIdFilter,
+    taskId: taskIdFilter,
+    tryNumber: tryNumberFilter,
+    user: userFilter,
+  });
+
+  return (
+    <HStack justifyContent="space-between">
+      <HStack flexWrap="wrap" gap={4}>
+        {/* Timestamp Range Filters */}
+        <Input
+          onChange={handleDateTimeChange(AFTER_PARAM)}
+          placeholder={translate("auditLog.filters.startDateTime")}
+          size="sm"
+          type="datetime-local"
+          value={formatDateTimeLocalValue(afterFilter)}
+          width="200px"
```

Comment:
Remove all the hardcodeed `width`. 

Remove the `flexWrap="wrap"` of. the hstack you should get this:
<img width="1895" height="495" alt="Screenshot 2025-08-11 at 12 16 10" src="https://github.com/user-attachments/assets/62d3d668-269e-42d8-a4fa-53fa4946b534" />